app/models/contact.rb

```diff
@@ -28,6 +28,14 @@ def needs_primary_email?
     ['Client', 'Volunteer'].include? contactable_type
   end
 
+  def validate_first_name?
+    contactable_type != 'Department' && contactable_type != 'Profile'
```

Comment:
> Sorry looks like I got it wrong in #115 ;-)

How. What should be validated, and what not? I thought we should un-restrict the thing for user.profile, in order to avoid exceptions thrown because of that?